### DISCUSSION TITLE
Withdrawn Star Bug

### DISCUSSION THREAD

**Discussion by 4Vic** [community/community#37096] [Repositories] [answered]
Hi, even though I've withdrawn my stars, it still shows 18. 

I've only given 6 stars at the moment. There is an bug here.

Or is this how this system works? Even if you cancel your stars, does the old number appear? 

![github-stars-bug](https://user-images.githubusercontent.com/40694758/197507857-8525e705-6c87-4c89-a48c-0497d242d4ea.png)

**Comment by 4Vic** [comment, marked as answer]
I gave a star to a different project and withdrew it. The problem is fixed.

If you encounter such an bug, give a project a star and withdraw it. Your problem will be solved.

**Comment by ossanna16** [comment]
Hi there @4Vic and welcome to our community! Thank you for asking a great question 🙂

**Reply by imee2211** [reply]
Title: Wrong Transfer to Contract Address on TON Network

Hello Bybit Support,

I mistakenly sent 27 USDT today (June 19, 2025) from my Bybit account to a contract address on the TON network, which is not eligible for deposits.

Transaction Details:

TXID: 12c2929292201f5a35ce7cde541f31d3887749d763ff9acc4251d3587bff10cb

Amount: 27 USDT

Network: TON

Date: June 19, 2025

Recipient Address (mistaken): EQCxE6mUtQJKFnGfaROTKO†1lZbDiiX1kCixRv7Nw2ld_sDs


Please check if the funds are still within the system or if there is any way they can be recovered. I appreciate your help.
